commons/src/lib.rs

```diff
@@ -0,0 +1,6 @@
+pub mod config;
+pub mod log_id;
+pub mod symbols;
+
+#[cfg(feature = "testing")]
```

Comment:
Or should we name the feature simply as the module itself?

```suggestion
#[cfg(feature = "test_runner")]
```